File: problems/missing-number/scaffold.java
import java.util.Scanner;

public class solution {
    public static int findMissingNumber(int[] arr) {
        // Your solution goes here
        return missingNumber;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        String[] arrayString = input.split(",");
        int[] array = new int[arrayString.length];

        for (int i = 0; i < arrayString.length; i++) {
            array[i] = Integer.parseInt(arrayString[i].trim());
        }

        int missingNumber = findMissingNumber(array);
        System.out.println(missingNumber);
    }
}

File: problems/missing-number/scaffold.cpp
#include <iostream>
#include <sstream>
#include <vector>

using namespace std;

int findMissingNumber(const vector<int> &arr)
{
    // Your solution goes here
    return missingNumber;
}

int main()
{
    string input;
    getline(cin, input);

    vector<int> array;
    stringstream ss(input);
    string num;
    while (getline(ss, num, ','))
    {
        array.push_back(stoi(num));
    }

    int missingNumber = findMissingNumber(array);
    cout << missingNumber << endl;

    return 0;
}